yorkie/src/main/kotlin/dev/yorkie/api/ChangeConverter.kt

```diff
@@ -19,13 +24,11 @@ internal fun List<PBChange>.toChanges(): List<Change> {
 }
 
 internal fun Change.toPBChange(): PBChange {
-    return PBChange.newBuilder().apply {
+    return change {
         id = this@toPBChange.id.toPBChangeID()
-        message = this@toPBChange.message
-        this@toPBChange.operations.toPBOperations().forEachIndexed { index, operation ->
-            setOperations(index, operation)
-        }
-    }.build()
+        message = this@toPBChange.message ?: ""
```

Comment:
pass empty string if `Change.message` is null
(related discussion: https://github.com/yorkie-team/yorkie-ios-sdk/pull/24#discussion_r1004256228)